R/quantity.R

```diff
@@ -0,0 +1,34 @@
+#' @title Quantity
+#' @docType class
+#' @description  A quantity of the model (with unit, value) such as a Parameter or an Amount
+#'
+#' @field value The value of the quantity
+Quantity <- R6Class(
+  "Quantity",
+  inherit = Entity,
+  active = list(
+    value = function(value) {
+      private$wrapProperties("Value", value)
+    },
+    displayUnit = function(value) {

```

Comment:
Big need to add documentation to this but I want to discuss the API before continuing
A quantity in OSPSUite.Core (molecule, amount, parameter etc..) has  a value, dimension and baseunit
Then it also has a display unit and a value in display unit

I have exposed all those methods . Maybe we should only work with baseunit an value in base unit. But it's not as nice for display

Reply:
Wow, this greatly confuses me! I would only have the `unit` as base unit and `value` in base unit.

We could add a function to convert to base units (e.g. `convertToBaseUnit(dimension, unit, value)`)

Reply:
And also `convertFromBaseUnit(dimension, targetUnit, valueInBaseUnit)`

Reply:
I have removed the methods. They make sense if you think in OOP however